eldritch/mythos.py

```diff
@@ -471,13 +643,195 @@ def get_modifier(self, thing, attribute):
     return 0
 
 
+class Mythos46(Headline):
+  def __init__(self):
+    super().__init__("Mythos46", "Woods", "Society", {"moon"}, {"plus"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.append(events.ReleaseMonstersToLocation("Easttown", 2))
+    return seq
+
+
+class Mythos47(Headline):
+  def __init__(self):
+    super().__init__("Mythos47", "Graveyard", "Isle", {"circle"}, {"square", "diamond"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.append(events.ReleaseMonstersToLocation("Uptown", 2))
+    return seq
+
+
+class Mythos48(Headline):
+  def __init__(self):
+    super().__init__("Mythos48", "Isle", "Science", {"hex"}, {"slash", "triangle", "star"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.extend(ReturnAndIncrease({"Dimensional Shambler", "Hound"}))
+    return seq
+
+
+class Mythos49(Headline):
+  def __init__(self):
+    super().__init__("Mythos49", "Woods", "Society", {"hex"}, {"slash", "triangle", "star"})
+
+  def create_event(self, state) -> events.Sequence:
+    seq = super().create_event(state)
+    seq.events.extend([
+        events.CloseLocation("Administration", 1),
+        events.CloseLocation("Library", 1),
+        events.CloseLocation("Science", 1),
+        events.AddGlobalEffect(self, "mythos", state.turn_number + 1)
+        # Minor rare interaction with shuffling
+    ])
```

Comment:
Don't you need to include the trigger that puts it back into the deck at the end of the next mythos?